Makefile

```diff
@@ -67,6 +68,7 @@ ALL_FILES_WIN=\
     batavia\types\NotImplementedType.js \
     batavia\types\Range.js \
     batavia\types\Set.js \
+		batavia\types\SetIterator.js \
     batavia\types\Slice.js \
```

Comment:
This looks like more tab fun... I suspect the fix here is actually to correct all the *other* lines to actually use a tab, given make's general preference for tabs over spaces...